dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporterTest.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2020 Dynatrace LLC
+ * Copyright 2021 Dynatrace LLC
```

Comment:
I think this should be the year when it was initially released, so it stays 2020.